dbtemplates/utils/cache.py

```diff
@@ -1,20 +1,23 @@
 import django
-from django.core import signals
+from dbtemplates.conf import settings
 from django.contrib.sites.models import Site
+from django.core import signals
 from django.template.defaultfilters import slugify
 
-from dbtemplates.conf import settings
-
 
 def get_cache_backend():
     """
     Compatibilty wrapper for getting Django's cache backend instance
     """
-    if django.VERSION[0] >= 3 and django.VERSION[1] >= 2:
+    if (django.VERSION[0] >= 3 and django.VERSION[1] >= 2) or django.VERSION[
+        0
+    ] >= 4:  # noqa
```

Comment:
`(django.VERSION[0] >= 3 and django.VERSION[1] >= 2)` is true for 3.2+ and also 4.2+ and 5.2+ etc.

Not a problem in this case, but it's good practice to compare to tuples instead (and we can skip the second part of the OR):

```suggestion
    if (django.VERSION >= (3, 2):
```

_But_ we only support Django 3.2+ so we can remove this whole check and simplify the function.

It's a compatibility wrapper, so we can probably remove the whole function.

Reply:
As Django 3.2 is the lowest supported version I decided I'll remove it.